lib/hammer_cli_foreman/audit.rb

```diff
@@ -0,0 +1,36 @@
+module HammerCLIForeman
+
+  class Audit < HammerCLIForeman::Command
+
+    resource :audits
+
+    class ListCommand < HammerCLIForeman::ListCommand
+
+      output do
+        field :id, _("Id")
+        field :created_at, _("At")
+        field :remote_address, _("IP")
+        field :user_name, _("User name")
+        field :action, _("Action")
+        field :auditable_type, _("Audit type")
+        field :auditable_name, _("Audit name")
+      end
+
+      build_options
+    end
+
+
+    class InfoCommand < HammerCLIForeman::InfoCommand
```

Comment:
So api returns `audited_changes` for each audit. It's a hash where keys are the changed attributes and values are either a new value or array of [old, new] values, depending on whether it was an update or creation.

For example:
```ruby
"audited_changes" => {
  "name" => "subtest",
  "ignore_types" => [
    "ProvisioningTemplate",
    "Hostgroup"
  ],
  "description" => "",
  "label" => nil,
  "ancestry" => "2"
}
```

Therefore I would say we need to pre-process the api output first, to move values into some more predictable hashes. In hammer you can use method `extend_data` that gets the api data as a parameter. Something like:
```ruby
def extend_data(audit)
  audit["changes"] = audit["audited_changes"].map do |attribute, change|
    if change.is_a?(Array)
      {
        'attribute' => attribute,
        'old' => change[0],
        'new' => change[1]
      }
    elsif !change.nil?
      {
        'attribute' => attribute,
        'value' => change
      }
    end
  end.compact
  audit
end
```
...or some other modification based on how we want the output to look like.

I did some experiments and following output is the best I've been able to achieve so far:
example 1:
```
Audited changes: 
 1) Attribute: name
    Value:     tst
 2) Attribute: ancestry
    Value:     3
 3) Attribute: grub_pass
    Value:
 4) Attribute: lookup_value_matcher
    Value:     hostgroup=tst2/tst
```
example 2:
```
Audited changes: 
 1) Attribute: default
    Old:       
      --- on_demand
      ...
    New:       
      --- immediate
      ...
```

The field definition I used for that is:
```ruby
output ListCommand.output_definition do
  collection :changes, _("Audited changes") do
    field :attribute, _("Attribute")
    field :value, _("Value"), Fields::Field, :hide_blank => true
    field :old, _("Old"), Fields::LongText, :hide_blank => true
    field :new, _("New"), Fields::LongText, :hide_blank => true
  end
end
```
I didn't invest too much time into that so there are probably more readable ways of formatting the changes.